File: Oxygene/Island/Delphi VCL/Sample-Windows/Unit2.pas
namespace Sample;

interface

uses
  RemObjects.Elements.RTL.Delphi, 
  RemObjects.Elements.RTL.Delphi.VCL;

type
  TForm2 = public class(TForm)
  public
    Button1: TButton;
    Edit1: TEdit;
    Label1: TLabel;
    CheckBox1: TCheckBox;
    RadioButton1: TRadioButton;
    Button2: TButton;
    Button3: TButton;
    Edit2: TEdit;
    ListBox1: TListBox;
    procedure Button2Click(Sender: TObject);
    procedure Button1Click(Sender: TObject);
    procedure Button3Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Form2: TForm2;

implementation

procedure TForm2.Button1Click(Sender: TObject);
begin
  Label1.Caption := Edit1.Text;
end;

procedure TForm2.Button2Click(Sender: TObject);
begin
  ShowMessage('Hi there!');
end;

procedure TForm2.Button3Click(Sender: TObject);
begin
  ListBox1.Items.Add(Edit2.Text);
end;

end.

File: Silver/Island/Delphi VCL/Sample-Windows/Unit2.swift
import RemObjects.Elements.RTL.Delphi
import RemObjects.Elements.RTL.Delphi.VCL

var Form2: TForm2?

public class TForm2 : TForm {
	public __field Button1: TButton!
	public __field  Edit1: TEdit!
	public __field  Label1: TLabel!
	public __field  CheckBox1: TCheckBox!
	public __field  RadioButton1: TRadioButton!
	public __field  Button2: TButton!
	public __field  Button3: TButton!
	public __field  Edit2: TEdit!
	public __field  ListBox1: TListBox!

	public func Button1Click(Sender: TObject) {
		Label1.Caption = Edit1.Text
	}

	public func Button2Click(Sender: TObject) {
		ShowMessage("Hi there!")
	}

	public func Button3Click(Sender: TObject) {
		ListBox1.Items.Add(Edit2.Text)
	}
}